docs/network-flow-visibility.md

```diff
@@ -43,6 +47,51 @@ For visualizing the network flows, Antrea monitors the flows in Linux conntrack
 module. These flows are converted to flow records, and then flow records are post-processed
 before they are sent to the configured external flow collector.
 
+## Getting Started
+
+### Prerequisites
+
+To get started, make sure you have installed Antrea with the Flow Exporter
+feature enabled by setting the following `antrea-agent` config parameter in
+the Antrea deployment manifest.
+
+```yaml
+  antrea-agent.conf: |
+    ...
+    featureGates:
+      ...
+      FlowExporter: true
+```
+
+### Theia Installation
+
+To enable both Grafana Flow Collector and
+[NetworkPolicy Recommendation](networkpolicy-recommendation.md), please clone
+the theia repository, and install Theia and Flow Aggregator by runnning the
```

Comment:
Very pedant comment: since you've added `git clone` to the list of commands, it's not needed to state you need to clone the repo.

Reply:
Removed, thanks!